libfabric.spec

```diff
@@ -50,6 +51,10 @@ BuildRequires: autoconf, automake, libtool
 libfabric provides a user-space API to access high-performance fabric
 services, such as RDMA.
 
+%if 0%{?suse_version}
+Requires: %{suse_libname}%{?_isa} = %{version}-%{release}
+%endif
+
```

Comment:
This was supposed to go after line 48 (after the blank line, 49 to be more precise).  The [rpmlint warnings](https://jenkins-1.wolf.hpdd.intel.com:9090/blue/rest/organizations/jenkins/pipelines/daos-stack/pipelines/libfabric/branches/PR-66/runs/12/nodes/184/steps/374/log/?start=0) that made the Leap 15.4 RPM build stage unstable (yellow) are indicating the same thing.